sdk/core/core-client/src/interfaces.ts

```diff
@@ -559,3 +560,50 @@ export interface CommonClientOptions extends PipelineOptions {
    */
   allowInsecureConnection?: boolean;
 }
+
+export { createSpanFunction, CreateSpanFunctionArgs, Span };
```

Comment:
Another thing is I was trying to decide whether to re-export these from core-rest-pipeline or core-client (I don't think either depends on the other)... unfortunately both need core-tracing so either way both have to continue depending on core-tracing... but core-client felt like the right place

Reply:
+1 on it being `core-client`